Kms/src/V1/Gapic/KeyManagementServiceGapicClient.php

```diff
@@ -2225,12 +2241,12 @@ public function setIamPolicy($resource, $policy, array $optionalArgs = [])
 
     /**
      * Returns permissions that a caller has on the specified resource. If the
-     * resource does not exist, this will return an empty set of
-     * permissions, not a `NOT_FOUND` error.
-     *
-     * Note: This operation is designed to be used for building
-     * permission-aware UIs and command-line tools, not for authorization
-     * checking. This operation may "fail open" without warning.
+    resource does not exist, this will return an empty set of
+    permissions, not a `NOT_FOUND` error.
+
+    Note: This operation is designed to be used for building
+    permission-aware UIs and command-line tools, not for authorization
+    checking. This operation may "fail open" without warning.
```

Comment:
nit, but this shouldn't be happening and it's a little worry-some.

cc @noahdietz